_studio/mfx_lib/shared/src/libmfxsw_encode.cpp

```diff
@@ -79,6 +79,7 @@ struct CodecKey {
 
 struct Handlers {
     struct Funcs {
```

Comment:
Why do we need this [Funcs] struct? Might all that [ctor\query] be defined at [Handlers] directly?

Reply:
We need primary handlers and same set of handlers to support fallback. Certainty, we may repeat all 6 in rows, but to me having separate cases is more structured.